packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -128,8 +138,8 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
   private _onRenderCoin = (props: IPersonaCoinProps): JSX.Element | null => {
     const { coinSize, styles, imageUrl, imageAlt, imageShouldFadeIn, imageShouldStartVisible, theme, showUnknownPersonaCoin } = this.props;
 
-    // Render the Image component only if an image URL is provided
-    if (!imageUrl) {
+    // Do not render the Image component if either an image URL is not provided or we wish to show the initials when hovering over the image
+    if (!imageUrl || (this.props.showInitialsOnHover && this.state.isHovered)) {
```

Comment:
It seems like here we check if the `showInitialsOnHover` and the `isHovered` state is set, but we never add the handlers when `showInitialsOnHover` is not defined, so one of them seems redundant?. 

Perhaps we can simplify the this logic and remove `this.props.showInitialsOnHover` from here or change the `showInitialsOnHover ? this._onCoinMouseEnter : undefined` above into just ` this._onCoinMouseEnter` ?

Maybe it would be nice to move the `this.props.showInitialsOnHover` into the handlers and change this like to just `!imageUrl || this.state.isHovered`.

Reply:
Agree with your comments on this change. pulled the this.props.showInitialsOnHover change into the handlers and reduced the redundant code. Had to update the snapshots anytime persona is used however, due to handlers now always being used by persona coin.